docs: re-add `Servers tutorial` and change code snippet

animeshk923 opened the pull request:
## Description

- Re-added the Servers tutorial in the next-major-spec branch.
- Changed the code snippet to match v3.

**Related issue(s)**

Fixes #1859

animeshk923 committed:
re-add `Servers tutorial` and change code snippet

changes:

- Re-added the `Severs tutorial` in the next-major-spec branch.
- Changed the code snippet to match v3.

derberg reviewed (dismissed):
@AnimeshKumar923 perfectly updated! 👏🏼 

@alequetzalli you can approve and merge. Only example had to be changed. Animesh re-added existing tutorial and only modified the example - text unchanged

![Screenshot 2023-11-08 at 17 09 21](https://github.com/asyncapi/website/assets/6995927/cf6efb01-6ec7-45c1-9e0f-8c0595178080)

animeshk923 commented:
> @AnimeshKumar923 perfectly updated! 👏🏼

Thank you! My pleasure Sir! ![salute-emoji](https://github.com/asyncapi/website/assets/99868037/f249ab7f-02af-4393-a7c7-b2556611eb78)

derberg commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,59 @@
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
```
```suggestion
<CodeBlock highlightedLines={[6,7,8,9,10]}>
{`asyncapi: 3.0.0
info:
  title: Hello world application
  version: '0.1.0'
servers:
  production:
    host: broker.mycompany.com
    protocol: amqp
    description: This is "My Company" broker.
channels:
  hello:
    address: 'hello'
    messages:
      sayHelloMessage:
        payload:
          type: string
          pattern: '^hello .+$'
operations:
  receiveHello:
    action: 'receive'
    channel:
      $ref: '#/channels/hello'`}
</CodeBlock>
```

 CodeBlock do not like empty spaces

animeshk923 replied:
> CodeBlock do not like empty spaces 

Understood :+1: 

resolved with 780b7e0

animeshk923 committed:
Update pages/docs/tutorials/getting-started/servers.md

applied suggestion from: https://github.com/asyncapi/website/pull/2275#discussion_r1387723300

Co-authored-by: Lukasz Gornicki <lpgornicki@gmail.com>

quetzalliwrites commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,56 @@
+
+You've now added a new section called `servers` in your AsyncAPI document.
+
+You might have noticed that our example mentions `amqp`. This protocol is very common and was popularized by RabbitMQ (among others). We picked `amqp` for our example, but you can use any protocol. The most common protocols used are `mqtt` (widely adopted by the Internet of Things and mobile apps), `kafka` (popular for its streaming solution), `ws` (WebSockets are frequently used in browsers), an ...
```
```suggestion
You might have noticed that our example mentions `amqp`, a very common protocol that was popularized by RabbitMQ (among others). While our example uses `amqp`, you can use any protocol. The most common protocols used are `mqtt` (widely adopted by the Internet of Things and mobile apps), `kafka` (popular for its streaming solution), `ws` (WebSockets are frequently used in browsers), and `http` (used in HTTP streaming APIs).
```

quetzalliwrites commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,56 @@
+
+The `servers` section defines where your application should connect to start sending and receiving messages. 
+
+1. If you are using a <a href="https://dev.to/fmvilas/event-driven-architectures--asyncapi-db7" target="_blank" className="text-secondary-600 font-medium hover:underline">broker-centric architecture</a> such as Kafka or RabbitMQ, usually you specify the URL of the broker. 
```
```suggestion
1. If you are using a broker-centric architecture such as Kafka or RabbitMQ, specify the broker URL. 
```

not professional to link to a devto blog post, removing link

quetzalliwrites commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,56 @@
+
+## Conclusion
+
+Now you know where `Hello world application` connects to and you can start receiving `hello {name}` messages.
```
```suggestion
Now you know where the `Hello world application` connects to, and you can start receiving `hello {name}` messages.
```

quetzalliwrites commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,56 @@
+
+Now you know where `Hello world application` connects to and you can start receiving `hello {name}` messages.
+
+In the next chapter, you'll learn how to add security requirements to your server.
```
```suggestion
In the next section, you'll learn how to add security requirements to your server.
```

quetzalliwrites commented on pages/docs/tutorials/getting-started/servers.md:
```diff
@@ -0,0 +1,56 @@
+
+In the previous lesson, you learned how to create the definition of a simple [Hello World application](/docs/getting-started/hello-world). Let's take it from there.
+
+In this article, you'll learn how to add `servers` to your AsyncAPI document. Adding and defining servers is useful, because it specifies where and how to connect. The connection facilitates where to send and receive messages.
```
```suggestion
In this tutorial, you'll learn how to add `servers` to your AsyncAPI document. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
```

Commit:
tw editorial fixes

quetzalliwrites reviewed (approved):
@derberg, i think we need you to hit approve again? yes? or no? here I go and see what happens 😂

quetzalliwrites commented:
/rtm

quetzalliwrites commented:
oh nvm this one didnt re request a review for some reason O_O that makes no sense to me, but hey, it's merged!
